src/jetstream/plugins/metrics/kubernetes.go

```diff
@@ -0,0 +1,37 @@
+package metrics
```

Comment:
Should this be included upstream?

Reply:
It is for now - we could look at a separate metrics plugin for K8S.